src/codecoverage/backend/codecoverage_backend/v2/__init__.py

```diff
@@ -6,42 +6,60 @@
 from flask import abort
 
 from cli_common import log
-from codecoverage_backend.v2.base import NoResults
-from codecoverage_backend.v2.base import active_data
-from codecoverage_backend.v2.path import coverage_paths
+from codecoverage_backend.services.gcp import load_cache
 
+DEFAULT_REPOSITORY = 'mozilla-central'
 logger = log.get_logger(__name__)
 
 
-def coverage_for_path(path='', changeset=None):
+def coverage_latest(repository=DEFAULT_REPOSITORY):
+    '''
+    List the last 10 reports available on the server
```

Comment:
The frontend will likely only need the latest

Reply:
For now... I did that because it was easy to do now, and could provide a quick `<select>` on the frontend to display recent variations of a file's coverage